src/edu/csus/ecs/pc2/clics/API202306/AccountService.java

```diff
@@ -0,0 +1,149 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.ArrayList;
+
+import javax.inject.Singleton;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Feature;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.Provider;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.model.Account;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+import edu.csus.ecs.pc2.services.eventFeed.WebServer;
+
+/**
+ * WebService for handling accounts endpoint
+ * 
+ * @author John Buck
+ *
+ */
+@Path("/contests/{contestId}/{svc: accounts|account}")
+@Produces(MediaType.APPLICATION_JSON)
+@Provider
+@Singleton
+public class AccountService implements Feature {
+
+    private IInternalContest model;
+
+    private IInternalController controller;
+
+    public AccountService(IInternalContest inContest, IInternalController inController) {
+        super();
+        this.model = inContest;
+        this.controller = inController;
+    }
+
+    /**
+     * This method returns a representation of the current model teams in JSON format. The returned value is a JSON array with one team description per array element.
```

Comment:
The comment says that this method returns "teams", but it appears to me that the method will (can) return other types of accounts as well...

Reply:
Fixed in next push.